Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -46,6 +51,7 @@ final class PledgeDescriptionCell: UITableViewCell, ValueCell {
       |> ksr_constrainViewToEdgesInParent()
 
     self.configureStackView()
+    self.bindViewModel()
```

Comment:
Note-to-self that we're going to find a way to override `UITableViewCell` initializers to not have to do this.